### PR TITLE
HFI PDF Formatting changes

### PR DESCRIPTION
**Pull request by conbrad**
- Approved by @tessmgood offline if a printer is happy with it.
# Test Links:
[Percentile Calculator](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=1108&f=c5&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN#state=2ec784ca-c46a-49d0-b2b3-1cf32a9015a2&session_state=7d9447c8-db66-4661-b4cb-03d2ac0d1d8f&code=32292df4-2bdf-4f90-a4a8-c8dbcda682a9.7d9447c8-db66-4661-b4cb-03d2ac0d1d8f.2b63f390-f3dc-43ae-89f2-016453863476)
[Fire Behaviour Advisory](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-1807.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

**Review by Sybrand — CHANGES_REQUESTED** [review]

File: api/app/hfi/pdf_generator.py [hunk 1]
```diff
@@ -32,11 +32,14 @@ def generate_pdf(result: HFIResultResponse,
                                       fire_centre_name,
                                       jinja_env)
 
-    left_footer = f'Exported on {datetime_generated.isoformat()} by {idir}'
+    left_footer = f'Exported on {datetime_generated.isoformat()} by {idir} | https://psu.nrs.gov.bc.ca/hfi-calculator'
     options = {
-        'page-size': 'Tabloid',
+        'page-size': 'A4',
```
[the remark this answers is not in the document]
Comment (conbrad) [reply api/app/hfi/pdf_generator.py]: Changed to letter

**Review by Sybrand — APPROVED** [review]

**Review by andrea-williams — APPROVED** [review]

File: api/app/hfi/templates/daily_template.jinja.html [hunk 2]
```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <title>HFI Daily</title>
+        <title>BC Wildfire Services - HFI Calculator - {{fire_centre_name}} - Date: {{date}}</title>
```
[the remark this answers is not in the document]
Comment (andrea-williams) [reply api/app/hfi/templates/daily_template.jinja.html:4]: it's my pleasure to be of service(s)

[the remark this answers is not in the document]
Comment (conbrad) [reply api/app/hfi/templates/daily_template.jinja.html]: Updated

File: api/app/hfi/templates/prep_template.jinja.html [hunk 3]
```diff
@@ -1,16 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>Prep Cycle Template</title>
+    <title>
+      BC Wildfire Services - HFI Calculator - {{ fire_centre_name }} - Prep
```
[the remark this answers is not in the document]
Comment (conbrad) [reply api/app/hfi/templates/prep_template.jinja.html]: Updated

**Review by JulianForeman — APPROVED** [review]

**Pull request merged**